byterun/gc_ctrl.c

```diff
@@ -48,6 +48,8 @@ extern uintnat caml_major_heap_increment; /* percent or words; see major_gc.c */
 extern uintnat caml_percent_free;         /*        see major_gc.c */
 extern uintnat caml_percent_max;          /*        see compact.c */
 extern uintnat caml_allocation_policy;    /*        see freelist.c */
+extern value caml_ephe_none;              /*        see weak.c */
+extern gc_phase_t caml_gc_phase;          /*        see major_gc.c */
```

Comment:
I don't think this `extern` decl is needed, you get it from `major_gc.h`

Reply:
Removed them.